examples/gen-ai-fastchat/README.md

```diff
@@ -85,19 +85,20 @@ After **terraform apply** completes, wait about 10 mins. During this time, the A
 ## Running the Demo
 1. As mentioned above, **wait ~10 minutes** for the Recipe to download/install FastChat and the LLM model before continuing
 2. SSH into newly created AWS EC2 instance. 
-3. The terraform module creates a key pair and adds the public key to the EC2 instance. It keeps the private key in the same folder from where the **terraform apply** was run.
+3. The terraform module creates a key pair and adds the public key to the EC2 instance. It keeps the private key in the same folder from where the **terraform apply** was run. (tf.private)
```

Comment:
I think the filename is tfkey.private

Reply:
i updated that.  After Talking to Lucas he wants me to write the Demo for AWS Cloudshell and see if we can simplify the instructions.  I cant test until the module is updated with the AMI code changes.